machine/accton/accton_as5610_52x/u-boot/platform-accton-as5610_52x.patch

```diff
@@ -1895,9 +1972,35 @@ index 0000000..3e0924d
 +				strmhz(buf, ddr_data_rate));
 +
 +	if(ddr_data_rate <= DATARATE_667MHZ)
-+		memcpy(&ddr_cfg_regs, &ddr_cfg_regs_667, sizeof(ddr_cfg_regs));
++	{
++		if( *((volatile unsigned char*)(CONFIG_SYS_CPLD_BASE)) < 0x10)
```

Comment:
so I understand -- you are reading a CPLD register (presumably a platform version register or something) to decide which DDR parameters to use?  Seems OK, just trying to understand.